docs/Reference/Rest_API/Rest.md

```diff
@@ -61,6 +61,23 @@ You can also use tools such as [Postman] or [cURL] to interact with Teku APIs.
         }
         ```
 
+## Enable the validator client API
+
+The [validator client API methods](https://consensys.github.io/teku/#tag/Validator) are enabled separately from the REST API methods.
```

Comment:
this is actually the keymanager api that is enabled on the VC, or potentially the BN. Hopefully at some point should be available at https://consensys.github.io/teku/keymanager-APIs/
@NicolasMassart was looking at publishing to there...

External standard reference is: https://ethereum.github.io/keymanager-APIs/